src/webapp/reports/mal-data-subscription/DataSubscriptionViewModel.ts

```diff
@@ -1,33 +1,58 @@
+import { NamedRef } from "../../../domain/common/entities/Base";
 import { Config } from "../../../domain/common/entities/Config";
 import {
-    MalDataSubscriptionItem,
-    SubscriptionStatus,
-    getDataSubscriptionItemId,
-    getSubscriptionValue,
+    DashboardSubscriptionItem,
+    DataElementsSubscriptionItem,
+    getDashboardSubscriptionItemId,
+    getDataElementSubscriptionItemId,
 } from "../../../domain/reports/mal-data-subscription/entities/MalDataSubscriptionItem";
 
-export interface DataSubscriptionViewModel {
+export interface DataElementSubscriptionViewModel {
     id: string;
     dataElementName: string;
     dataElementId: string;
-    sectionName: string;
+    section: NamedRef | undefined;
     subscription: boolean;
     lastDateOfSubscription: string;
 }
 
-export function getDataSubscriptionViews(
+export interface DashboardSubscriptionViewModel {
+    id: string;
+    name: string;
+    subscribedElements: string;
+    subscription: string;
```

Comment:
Here instead of only `string` maybe could be a specific type `"Subscribed" | "Not Subscribed" | "Subscribed to some elements"` like you do in `MalDataSubscriptionItem.ts`